.github/workflows/test.yml

```diff
@@ -224,6 +227,54 @@ jobs:
           targets: test:e2e
           projects: ${{matrix.projectName}}
 
+  test_e2e_ee:
+    name: Test E2E EE
+    runs-on: ubuntu-latest
+    needs: [get-affected]
+    if:  ${{ fromJson(needs.get-affected.outputs.test-e2e-ee)[0] }}
+    timeout-minutes: 80
+    strategy:
+      # One job for each different project and node version
+      matrix:
+        projectName: ${{ fromJson(needs.get-affected.outputs.test-e2e-ee) }}
+    permissions:
+      contents: read
+      packages: write
+      deployments: write
+      id-token: write
+    steps:
+      - run: echo ${{ matrix.projectName }}
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+          token: ${{ secrets.SUBMODULES_TOKEN }}
+      - uses: ./.github/actions/setup-project
+      - uses: ./.github/actions/setup-redis-cluster
+      - uses: mansagroup/nrwl-nx-action@v3
+        name: Lint and build
+        with:
+          targets: lint,build
+          projects: ${{matrix.projectName}}
+
+      - uses: ./.github/actions/start-localstack
+
+      - uses: ./.github/actions/run-worker
+        if: ${{matrix.projectName == '@novu/api' }}
```

Comment:
Is this test suite planned to be executed for multiple apps? That means that the EE tests will live in API, Worker, WS, etc?

Reply:
For now, they will live in the consumer app, in this case it will be the API